docs/source/search-syntax.rst

```diff
@@ -80,21 +83,50 @@ For example:
 Run Attributes
 ~~~~~~~~~~~~~~
 
-The search syntax supports searching runs using two attributes: ``status`` and ``artifact_uri``. Both attributes have string values. Other fields in :py:class:`mlflow.entities.RunInfo` are :ref:`system_tags` that are searchable using the UI and the API. The search returns an error if you use other attribute names in the filter string. 
+You can search using two run attributes contained in :py:class:`mlflow.entities.RunInfo`: ``status`` and ``artifact_uri``. Both attributes have string values. 
+
+Other fields in ``mlflow.entities.RunInfo`` are :ref:`MLflow tags <system_tags>` that are searchable as ``tag`` identifiers (see :ref:`mlflow_tags` below); search returns an error if you use these names in the ``attributes`` identifier. 
```

Comment:
```
:ref:`MLflow tags <system_tags>
```

Does this render correctly as a link?

Reply:
There's a backtick at the end, so yes it's a good link.